[world-vercel] Send remoteRefBehavior=lazy on v4 metadata-only event listings (#2415)
vercel/workflow: .changeset/v4-lazy-list-refs.md

---
"@workflow/world-vercel": patch
---

> Skip transferring event payload bytes for `resolveData: 'none'` listings on the v4 wire.

Author: ```suggestion
Skip transferring event payload bytes when listing events with `resolveData: 'none'` using the v4 API.
```
